### DISCUSSION TITLE
How do I restore a Plex backup?

### DISCUSSION THREAD

**Discussion by rifatozkancomtr** [sachinsenal0x64/fixarr-old#7] [Q&A]
How do I restore a Plex backup?

**Comment by sachinsenal0x64** [comment]
Hey, sorry for the late reply. i will implement this feature asap. for now, you have to manually replace the backup folder.


```
Operating System | Location

Windows| %LOCALAPPDATA%\Plex Media Server
MacOS|   ~/Library/Application Support/Plex Media Server/
Linux|   $PLEX_HOME/Library/Application Support/Plex Media Server/ or /var/lib/plexmediaserver/Library/Application Support/Plex Media Server/
ASUSTOR| /volume1/Plex/Library

```
[For More Details](https://support.plex.tv/articles/202915258-where-is-the-plex-media-server-data-directory-located/)
